bin/commands/init/vsam/.errors

```diff
@@ -1,3 +1,5 @@
+ZWEL0148E|148|%s (%s) is not defined in Zowe YAML configuration file.
```

Comment:
it's interesting to see the error message template implemented this way... given that it is the exact same as 149 & 157, it makes me wonder if they should indeed be the exact same message. what do you think?

Reply:
Yes, I was also concerned about that, and maybe you're right, they could be merged into a single message. Although is also true that the causes are different for all the 3 messages.

Reply:
But the failure cause is different, isn't it? I guess that it makes sense to keep them under different msg ID, but maybe I miss something

Reply:
1) Agree with @dkelosky - this is not a good practice, there is already an error message 157, which is generic and can be used for different problems.
2) This pull request [#3718](https://github.com/zowe/zowe-install-packaging/pull/3718) will remove the shell version  of this command anyway.

Reply:
Thanks @Martin-Zeithaml  for the notice about the other PR. Do you know how these messaged will be handled by these changes? Cause I will revert now the changes, but I'm not sure how these kind of messages like:
"Error ZWEL0157E: Zowe Caching Service VSAM data set Non-RLS volume (zowe.setup.vsam.volume) is not defined in Zowe YAML configuration file.
will be then displayed

Reply:
[Like this](https://github.com/zowe/zowe-install-packaging/blob/2188d7db55adde994b5677da01615e81f771001c/bin/commands/init/vsam/index.ts#L45-L51).